bower.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "bootstrap",
   "description": "The most popular front-end framework for developing responsive, mobile first projects on the web.",
-  "version": "3.3.4",
```

Comment:
FYI: I had in my bower file v3.3.* and the removal of version broke my build using grunt wiredep, for anyone else that might have come across the same issue. It's ok now, I don't need the version in the bower file. I just locked down the specific version and it works. Out of curiosity, may I ask why remove the version in the bower file?